app/about/page.tsx

```diff
@@ -116,7 +116,7 @@ export default function About() {
           />
           <Container className="text-center mb-8">
             <Typography variant="body1" className="text-mb font-semibold">
-              NAN
+              Meave (something)
```

Comment:
You should ask Swan for her last name, instead of leaving it as (something), you don't want to push (something) to the website